docs/fundamentals/code-analysis/style-rules/ide0047-ide0048.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "IDE0047 and IDE0048: Parentheses preferences"
+description: "Learn about code analysis rules IDE0047 and IDE0048: Parentheses preferences"
+ms.date: 09/30/2020
+ms.topic: reference
+f1_keywords:
+- IDE0047
+- IDE0048
+- dotnet_style_parentheses_in_arithmetic_binary_operators
+- dotnet_style_parentheses_in_relational_binary_operators
+- dotnet_style_parentheses_in_other_binary_operators
+- dotnet_style_parentheses_in_other_operators
+helpviewer_keywords:
+- IDE0047
+- IDE0048
+- dotnet_style_parentheses_in_arithmetic_binary_operators
+- dotnet_style_parentheses_in_relational_binary_operators
+- dotnet_style_parentheses_in_other_binary_operators
+- dotnet_style_parentheses_in_other_operators
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# Parentheses preferences (IDE0047 and IDE0048)
+
+|Property|Value|
+|-|-|
+| **Rule ID** | IDE0047 and IDE0048 |
+| **Title** | IDE0047: Remove unnecessary parentheses<br/> IDE0048: Add parentheses for clarity |
+| **Category** | Style |
+| **Subcategory** | Language rules |
+| **Applicable languages** | C# and Visual Basic |
+| **Introduced version** | Visual Studio 2017 version 15.8 |
+
+## Overview
+
+The style rules in this section concern parentheses preferences, including the use of parentheses for arithmetic, relational, and other binary operators.
+
+## Options
+
```

Comment:
Would be nice to add an index to the options here since there are quite a few. Another "option" would be to remove the Options H2 and promote each individual option to an H2, in which case they'd be automatically indexed in the "In this article" section. I'm actually leaning towards this idea.

Reply:
Sounds good, let me make these changes.

Reply:
Done.